qcog_python_client/qcog/pytorch/validate/validatehandler.py

```diff
@@ -24,6 +30,7 @@ class ValidateHandler(Handler):
     }
 
     async def handle(self, payload: ValidateCommand) -> UploadCommand:
+        """Handle the validation."""
         validated: list = []
```

Comment:
I still don't think this is used

Reply:
No, it's not, but eventually I have plan to use it. For example, the validation of the trianing function returns the current arguments of the function and the expected types. Pulling it from the list of the validated file, can be used to dispatch a command to validate the parameters uploaded by the user